components/gitpod-db/src/typeorm/entity/db-workspace-instance.ts

```diff
@@ -117,4 +118,10 @@ export class DBWorkspaceInstance implements WorkspaceInstance {
         transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED,
     })
     usageAttributionId?: string;
+
+    @Column({
+        type: "simple-json",
+        nullable: true,
```

Comment:
This does not work. Let's do it like in the examples above using:
```
    @Column({
        default: "",
        transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED,
    })
```
For the migration, just check what those for these fields.

Reply:
@geropl I wanted to apply this, but then it occurred to me that we'd actually get rid of any JSON type here, so we'd have to serialize and deserialize the values here?

I know you mentioned a pitfall you wanted to prevent here, and I remember changing something like this in the past, but the specifics are a bit hazy.

Reply:
Whoops, my bad. let me find a fitting example... :eyes:

Reply:
@filiptronicek Turns out we don't have single "proper" example in our code base... :facepalm: 

IMO - because the fundamental issue is that we can't convert `null` to `undefined` with typeorm - the only type safe way to do this is to always set a default value (of `{}`, for instance). But that's invasive, and does not fit here as well.

I suggest we just leave it with your approach, as we already do it the same way everywhere else. :+1:

Reply:
Oh wow, that's unfortunate. I hope that since it's marked as an optional property, accessing it is not too problematic.